packages/@coorpacademy-components/src/util/test/time-format.js

```diff
@@ -0,0 +1,17 @@
+import test from 'ava';
+import {formatMinutes} from '../time-format';
+
+test('should format minutes correctly', t => {
+  const minutes1 = 60;
+  const timeString1 = '1h ';
+
+  const minutes2 = 59;
+  const timeString2 = '59min';
+
+  const minutes3 = 61;
+  const timeString3 = '1h 01min';
+
```

Comment:
You can all variables. (Here they bring no particular value, while introducing some mental overhead)
You can just `t.is(format....., )`>

(There is way to factirze if you can, I can show you at some point).


Also, add some edge cases. Like 0, and 120. 
- 0 give something that is not necessary what we want (empty string), 
- 120 (and all exact hours) is buggy with a trialing space :space_invader: